src/expr/interpret/eval.hpp

```diff
@@ -479,7 +480,7 @@ class ExpressionEvaluator : public ExpressionVisitor<TypedValue> {
   }
 
   TypedValue Visit(Function &function) override {
-    FunctionContext function_ctx{dba_, ctx_->memory, ctx_->timestamp, &ctx_->counters, view_};
+    functions::FunctionContext<DbAccessor> function_ctx{dba_, ctx_->memory, ctx_->timestamp, &ctx_->counters, view_};
```

Comment:
I am not a big fun of `CTAD` so usually I avoid it. If you have a strong preference I am happy to use it, although I think it would fail in this particular case.

Reply:
I don't